src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -159,9 +158,16 @@ class WatershedFilterAlgo : public ITKFilter
 	double watershedLevel_val_;
 
 	std::vector<std::string> dst_layer_ids_;
-	std::vector<_LabelType> watershed_image_output_labels_;
-  
+	std::vector<WtrLabelType> watershed_image_output_labels_;
+
 public:
+
+	void report_debug_message( const std::string& msg )
```

Comment:
I was following the same methodology as context->report_error( msg );  I can take it out if you would like.